### PR TITLE
chore: bump prometheus-adapter

### PR DESCRIPTION
**Pull request by hectorj2f**
**What type of PR is this?**

Chore
**What this PR does/ why we need it**:

Bump prometheus adapter to latest, part of https://jira.d2iq.com/browse/D2IQ-72964
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by branden — APPROVED** [review]

File: addons/prometheusadapter/prometheusadapter.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: prometheusadapter
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-4"
     appversion.kubeaddons.mesosphere.io/prometheusadapter: "v0.7.0"
     values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/prometheus-community/helm-charts/a2b924bf7c439f0531634983dd941eeec2917725/charts/prometheus-adapter/values.yaml"
```
Comment (gracedo) [inline addons/prometheusadapter/prometheusadapter.yaml:11]: does the values need to be updated?

**Comment by branden** [comment]
My PR to bump the appversion was merged, want to pull that in with this PR? https://github.com/prometheus-community/helm-charts/pull/462

**Comment by hectorj2f** [comment]
Yes, @branden. That is what we wanted, i will update it today.

File: addons/prometheusadapter/prometheusadapter.yaml [hunk 2]
```diff
@@ -6,9 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: prometheusadapter
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-4"
     appversion.kubeaddons.mesosphere.io/prometheusadapter: "v0.7.0"
-    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/prometheus-community/helm-charts/a2b924bf7c439f0531634983dd941eeec2917725/charts/prometheus-adapter/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/branden/prometheus-helm-charts/57c87ea696d270dca5df866c3195d851956ff27f/charts/prometheus-adapter/values.yaml"
```
Comment (armandgrillet) [inline addons/prometheusadapter/prometheusadapter.yaml:11]: You should be able to use the standard repo: https://github.com/prometheus-community/helm-charts/commit/ef6391472c694c8cb1412b277f94ae3b7cfe972e

Comment (hectorj2f) [reply addons/prometheusadapter/prometheusadapter.yaml]: done

**Review by tillt — APPROVED** [review]

**Pull request merged**